cortex-ar/src/register/iciallu.rs

```diff
@@ -0,0 +1,35 @@
+//! Code for managing ICCIALLU (Instruction Cache Invalidate All to PoU)
+
+use crate::register::{SysReg, SysRegRead, SysRegWrite};
+
+/// ICC_PMR (*Interrupt Controller Interrupt Priority Mask Register*)
+pub struct Icciallu(pub u32);
+impl SysReg for IcciAllu {
```

Comment:
This module refers to Iccallu, IccAllu, and IccPmr. It probably won't even compile?

Reply:
I removed ICCIALLU for now. I started with one register, and then I saw that most of the cache maintenance ops are done in the vendor startup with direct inline assembler commands.
I think it might still make sense to add these registers later, but then as a separate PR with more of of them added.